docs/application/dotnet/guides/nui/visuals.md

```diff
@@ -1,390 +1,565 @@
+| `Color`               | `Color` | Yes      | The color of the visual.                                                                              |
+| `RenderIfTransparent` | `bool`  | No       | Specifies whether to render the visual in case of the `MixColor` being transparent (default `false`). |
 
-The color visual renders a solid color to the control's quad.
 
-**Figure: Color visual**
 
-![Color visual](media/color-visual.png)
+**Usage:**
+The following example illustrates how to use a `ColorVisual`:
 
-The following table lists the supported properties. The visual type is `Color` and the visual map is `ColorVisual`.
 
-**Table: Color visual properties**
+   ```csharp
+   ColorVisual _colorVisual_1 = new ColorVisual();
 
-| Property   | Type    | Required | Description         |
-|----------|-------|--------|-------------------|
-| `MixColor` |  `Vector4` | Yes      |  The color to be used. |
+   // Obligatory properties
+   _colorVisual_1.Color = Color.Red;
 
-**Usage:**
+   // Optional properties:
+   _colorVisual_1.Size = new Size2D(400,400);
+   _colorVisual_1.DepthIndex = 5;
+
+   _visualView.AddVisual("RedRectangle", _colorVisual_1);
 
-The following example illustrates how to use a property map for a color visual. The visual is created with the factory `CreateVisual()` method.
 
-```
-private const int PROPERTY_REGISTRATION_START_INDEX = 10001000;
-private const int ColorVisualPropertyIndex = PROPERTY_REGISTRATION_START_INDEX + 1;
-private const int PrimitiveVisualPropertyIndex = PROPERTY_REGISTRATION_START_INDEX + 2;
+   ColorVisual _colorVisual_2 = new ColorVisual();
 
-private VisualBase _colorVisual;
+   // Obligatory properties
+   _colorVisual_2.Color = new Color(0.0f, 0.0f, 1.0f, 0.8f);
 
-PropertyMap colorVisual = new PropertyMap();
-colorVisual.Add(Visual.Property.Type, new PropertyValue((int)Visual.Type.Color))
-           .Add(ColorVisualProperty.MixColor, new PropertyValue(_color));
-_colorVisual = VisualFactory.Get().CreateVisual(colorVisual);
+   // Optional properties:
+   _colorVisual_2.Size = new Size2D(300,300);
+   _colorVisual_2.Origin = Visual.AlignType.BottomEnd;
+   _colorVisual_2.AnchorPoint = Visual.AlignType.BottomEnd;
+   // Shifting the visual left and up 
+   _colorVisual_2.RelativePosition = new RelativeVector2(-0.1f, -0.1f);
+   // For values > 0 corners are rounded
+   _colorVisual_2.CornerRadius = 50.0f;
+   // Setting a value lower than for the previous object causes it to be covered
+   // Without changing this value, the objects are drawn in the order they were added
+   _colorVisual_2.DepthIndex = _colorVisual_1.DepthIndex - 1;
 
-RegisterVisual(ColorVisualPropertyIndex, _colorVisual);
+   _visualView.AddVisual("BlueRectangle", _colorVisual_2);
+   ```
 
-/// Set the color visual depth index
-_colorVisual.DepthIndex = ColorVisualPropertyIndex;
-```
 
-<a name="gradientvisual"></a>
-## Gradient Visual
+**Figure: ColorVisual after implementing example (_visualView background color set to gray).**
```

Comment:
I can do that, but please confirm, since I just removed all figures captions according to https://github.com/Samsung/tizen-docs/pull/1367#discussion_r538156296